server/queues/tasks/DeliverWebhookTask.ts

```diff
@@ -539,10 +539,13 @@ export default class DeliverWebhookTask extends BaseTask<Props> {
         "user-agent": `Outline-Webhooks${
           env.VERSION ? `/${env.VERSION.slice(0, 7)}` : ""
         }`,
+        "Outline-Signature": subscription.signature(
+          JSON.stringify(requestBody)
+        ),
       };
       response = await fetch(subscription.url, {
         method: "POST",
-        headers: requestHeaders,
+        headers: requestHeaders as any,
```

Comment:
I think we need to generate the signature and only add to `requestHeaders` if it's not undefined, should remove the requirement to cast to `any` here.